llvm/CMakeLists.txt

```diff
@@ -1150,6 +1150,11 @@ if (UNIX AND ${CMAKE_SYSTEM_NAME} MATCHES "AIX")
           add_compile_definitions(_XOPEN_SOURCE=700)
           add_compile_definitions(_LARGE_FILE_API)
 
+  # Make functions, like errno, thread safe.
+  if (LLVM_ENABLE_THREADS)
+    add_compile_definitions(_THREAD_SAFE)
```

Comment:
Why do we need to define this macro manually? Would using `-pthread` make more sense?

Reply:
Ok, I see, `-pthread` implies `_THREAD_SAFE` on AIX. Thanks for pointing it out.